src/chunk.h

```diff
@@ -18,6 +18,13 @@
 #include "hypertable.h"
 #include "export.h"
 
+/*
+ * Set FormData_chunk.compressed_chunk_id to AN_COMPRESSED_CHUNK_ID on the access
+ * node to mark a distributed chunk as compressed so that distributed compression
+ * policy can ignore it after compressing its replicas in the data nodes.
+ */
+#define AN_COMPRESSED_CHUNK_ID -1
```

Comment:
Strictly, I think it should be `A_COMPRESSED_CHUNK_ID`, but I would probably just do:

```suggestion
#define COMPRESSED_CHUNK_ID -1
```

which is also consistent with `INVALID_CHUNK_ID`

Reply:
I added `AN_` to denote that it's valid for Access Nodes. It shouldn't be used in Data Nodes. That was the rationale anyway.

Reply:
Just my few cents: It is clear in the comment that this is the purpose, so I do not think it necessarily need to be reflected in the name.

Reply:
Understood, we're probably going to use a flag for this anyway since @gayyappan is merging another conflicting PR.